tests/test_extractors.py

```diff
@@ -80,3 +92,53 @@ async def test_async_emulators() -> None:
         assert len(all_emu_mps_results[True].processed_data) == len(
             all_emu_mps_results[False].processed_data
         )
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("extractor", [RemoteQPUExtractor, RemoteEmuMPSExtractor])
+async def test_async_remote_qpu_extractor(extractor: Type[BaseRemoteExtractor]) -> None:
+    """
+    Test that the remote extractors can execute without exploding (both sync and async).
+    """
+    # Load dataset
+    original_ptcfm_data = [
```

Comment:
Note to self: this expression is pretty long to execute, so if we start doing it too many times, we might want to turn it into a fixture.